Makefile

```diff
@@ -1,46 +1,98 @@
 # Package configuration
 PROJECT = code-annotation
 COMMANDS = cli/server
-DEPENDENCIES = github.com/golang/dep/cmd/dep
+DEPENDENCIES = github.com/golang/dep/cmd/dep github.com/jteeuwen/go-bindata
 
 HOST ?= 127.0.0.1
 PORT ?= 8080
-SERVER_URL ?= //$(HOST):$(PORT)
+REACT_APP_SERVER_URL ?= //$(HOST):$(PORT) # frontend uses $(REACT_APP_SERVER_URL) as backend
+UI_DOMAIN ?= $(REACT_APP_SERVER_URL) # /oauth-callback redirects to $(UI_DOMAIN)/?token=__TOKEN__
+
+YARN_PRODUCTION ?= true
+
+# Tools
+YARN = yarn
+GODEP = dep
+GOLINT = golint
+GOVET = go vet
+BINDATA = go-bindata
+
+# ci variables
+TRAVIS_BUILD_DIR ?= $(shell pwd)
+CGO_ENABLED = 1
+PKG_OS = linux
+DOCKER_OS = linux
+DOCKER_ARCH = amd64
 
 # Including ci Makefile
 CI_REPOSITORY ?= https://github.com/src-d/ci.git
 CI_PATH ?= $(shell pwd)/.ci
 MAKEFILE := $(CI_PATH)/Makefile.main
 $(MAKEFILE):
-	git clone --quiet --depth 1 $(CI_REPOSITORY) $(CI_PATH);
+	@git clone --quiet --depth 1 -b v1 $(CI_REPOSITORY) $(CI_PATH);
 -include $(MAKEFILE)
 
-# set enviroment variables from .env file
-include .env
-export $(shell sed 's/=.*//' .env)
+# Set enviroment variables from .env file
+ENV ?= .env
+$(ENV):
+	touch $(ENV)
```

Comment:
I would prefer import `.env` only if it exists instead of creating new one. But up to you.

Reply:
Done :dagger: